src/makeRobot.js

```diff
@@ -39,6 +39,61 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    get info() {
+      return `name: ${this.name}, `
+        + `chip version: ${this.version}, `
+        + `wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(step = 1) {
+      if (step <= 0) {
+        return this;
+      }
+      this.coords.y += step;
```

Comment:
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }
```
можеш так всюди зробити,коду буде менше

Reply:
Мушу визнати - це слушо. Дякую :)

Reply:
Agree, gold rule `don't repeat yourself`